### DISCUSSION TITLE
Will there be a fix for the containers not showing up issue?

### DISCUSSION THREAD

**Discussion by Luxvao** [podman-desktop/podman-desktop#871] [Q&A]
It's clearly a very common issue for containers and or images to not show up in the gui. I'm aware of the restarting the podman machine fix, but as I have a local minikube cluster running it can be time consuming to stop the cluster, stop the machine and start it again. So is there any plans for a fix or something?

**Comment by benoitf** [comment]
@Luxvao when you don't see any containers do you have traces in view / toggle developer tools and then console
We're listening events from the socket so I'm interested to know if you see events, errors or not ?

**Reply by Luxvao** [reply]
I can't really replicate if it happens again, I'll make an issue about it I guess
